docs/site/Deploying_to_ibm_cloud_kubernetes.md

```diff
@@ -135,38 +100,27 @@ In the command window, stop the application with
 Ctrl + C
 ```
 
-## Step 6: Create a Dockerfile
-
-Let's get the application running in Docker.
+## Step 3: Build a Docker image
 
-In the main project directory, create a file named `Dockerfile` .
+Review the two Docker-related files that have been conveniently provided,
+`.dockerignore` and `Dockerfile`, but leave them unchanged for this tutorial.
 
-Place this inside:
+Notice the `HOST` and `PORT` environment variable values:
 
 ```
-FROM node:10.14-alpine
```

Comment:
The scaffolded Docker file starts with `FROM node:10`, while your original guide used the Alpine version. I am not very familiar with Docker, I believe Alpine images are smaller that regular ones? Anyhow, changes in this area are probably out of scope of this pull request.

Reply:
@bajtos , yes I agree that an alpine image is slimmer.
As it turns out, the IBM Cloud has a vulnerability scanner with regards to Docker images, and after I uploaded the recent image, I got a few security vulnerabilities and configuration issues. My tutorial has always had a link for addressing vulnerabilities by creating exemptions, and then everything works in kubernetes. I mentioned to Raymond yesterday that I had to create exemptions, and we decided to try to find a "clean" image so user would not have to go through that step. I was able to get rid of security vulnerabilities by moving from node:10 to node:10.15. But there was still configuration issues regarding /etc/mysql : 
```
The scan results show that 3 ISSUES were found for the image.

Configuration Issues Found
==========================

Configuration Issue ID                     Policy Status   Security Practice                                    How to Resolve
application_configuration:mysql.ssl-ca     Active          A setting in /etc/mysql/my.cnf that specifies the    ssl-ca is not specified in /etc/mysql/my.cnf.
                                                           Certificate Authority (CA) certificate.
application_configuration:mysql.ssl-cert   Active          A setting in /etc/mysql/my.cnf that specifies the    ssl-cert is not specified in /etc/mysql/my.cnf
                                                           server public key certificate. This certificate      file.
                                                           can be sent to the client and authenticated
                                                           against its CA certificate.
application_configuration:mysql.ssl-key    Active          A setting in /etc/mysql/my.cnf that identifies the   ssl-key is not specified in /etc/mysql/my.cnf.
                                                           server private key.
```
I found an IBM document ,https://github.com/IBM-Bluemix-Docs/tutorials/blob/master/continuous-deployment-to-kubernetes.md , which describes how to get rid of this problem (in a ubuntu image) , and essentially they do:
```
RUN apt-get remove -y mysql-common \
  && rm -rf /etc/mysql
```
in their Docker file.

In Alpine, apt-get is not used, but : apk . 

There didn't seem to be 'mysql-common' installed on the alpine image, or any package with string 'mysql', so I didn't need to do : apk del <some sql package>.

However the /etc/mysql directory does exist, and so I do remove it.

As for size, node:10.15 creates an image that is 424 MB, and node:10.15-alpine is 102MB.

I will create a different PR for changing the Dockerfile involved in 'lb4 app --docker' .

Reply:
See https://github.com/nodejs/docker-node#image-variants. If it's only for size, `slim` is the choice.

Reply:
I tried  node:10.15-slim and the image is 131MB vs node:10.15-alpine is 102 MB.
But the 'slim' had the added advantage that there is no /etc/mysql to remove.

Reply:
I spoke with @raymond, and he'd rather have the 'slim' since we don't need to remove the /etc/mysql directory in the Dockerfile commands.

Created this PR : https://github.com/strongloop/loopback-next/pull/2549

Reply:
Thank you @emonddr for the investigation. I agree with the conclusion to use the `slim` version.